index.bs

```diff
@@ -522,6 +526,162 @@ See also:
 * [Security and privacy are essential](https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy)
 * [What data does this specification expose to an origin?](https://www.w3.org/TR/security-privacy-questionnaire/#underlying-platform-data)
 
+<h3 id="support-non-fully-active">Support non-"fully active" documents</h3>
+
+After a user navigated away from a document,
+the document might stay around in a non-[=Document/fully active=] state,
+and might be reused when the user navigates back to the entry holding the document.
+In the past, many APIs have missed specifying support for non-[=Document/fully active=] documents,
+making them hard to support in various user agents' implementation of "back/forward cache" (BFCache).
+
+To avoid this happening with your API,
+you should specify support for non-[=Document/fully active=] documents by following these guidelines:
+
+<h4 id="gate-fully-active">Gate actions with [=Document/fully active=] checks</h4>
+
+When performing actions that might update the state of a document,
+be aware that the document might not be [=Document/fully active=]
+and is considered as "non-existent" from the user's perspective.
```

Comment:
Do we intend prerendering pages to be considered fully active?

In same-origin prerendering at least, we plan to give them access to storage/shared state, and they receive updates. So from that perspective, they should be considered fully active, but the text above suggests that any document that is "non-existent" to the user should be not fully active. cc @domenic 

This also affects how we deal with https://github.com/w3c/ServiceWorker/issues/1594 as the current plan is for the service worker to not see bfcached pages but does see prerendered pages.

Reply:
I discussed this with @domenic a while back (see [Google-internal link](https://docs.google.com/document/d/1E93GazamR7FP_w_uBHwMtGeixCeGeiiJGkijpUYHybs/edit?resourcekey=0-ZWf7pJL-rGTUvyMhXlvK_Q&disco=AAAAIPLiOCY)), and I think prerendered page will be fully active but has its "is prerendering" bit set or something set or something like that, since we expect it to behave differently than bfcached pages more often than not.

Perhaps we should add a note here that there are other kinds of documents that are "non-existent" but also "fully active", specifically mentioning prerendered pages and link to https://jeremyroman.github.io/alternate-loading-modes/ ?

Reply:
Thanks I see. Yep adding a note would address my question, thanks!

Reply:
Added, thanks!